premium/usage_test.go

```diff
@@ -287,15 +287,20 @@ func TestUsageService_Increase_WithMinimumUpdateDuration(t *testing.T) {
 
 	usageClient := newClient(t, apiClient, WithBatchLimit(0), WithMinTimeBetweenFlushes(30*time.Second))
 
-	for i := 0; i < 10000; i++ {
+	rows := 10000
+	for i := 0; i < rows; i++ {
 		err = usageClient.Increase(1)
 		require.NoError(t, err)
 	}
 	err = usageClient.Close()
 	require.NoError(t, err)
 
-	assert.Equal(t, 10000, s.sumOfUpdates(), "total should equal number of updated rows")
-	assert.Equal(t, 2, s.numberOfUpdates(), "should only update first time and on close if minimum update duration is set")
+	assert.Equal(t, rows, s.sumOfUpdates(), "total should equal number of updated rows")
```

Comment:
I was thinking we can change the test to something like this:

```go
// Number of updates should be between 1 and 2 depending if when close is called there are remaining rows to update
assert.GreaterOrEqual(t, 1, s.numberOfUpdates())
assert.LessOrEqual(t, 2, s.numberOfUpdates())
```

I think it's ok to test the side effect of the code, doesn't really matter why it's 1 or 2. Now the test logic depends on the correctness of `dataOnClose`, not a big issue though. So you can take it or leave it

Reply:
Will take a note for next time :)